sponsoring/index.md

```diff
@@ -28,7 +28,21 @@ At the moment we've multiple sponsors via GitHub. In addition to that, multiple
 companies sponsor our cloud/CI resources.
 
 <div class="row">
-  {% for sponsor in site.sponsors %}
+  {% for sponsor in site.sponsors -%}
+    {% if sponsor.status == "current" -%}
     {% include sponsor-card.html width=4 vendor=sponsor %}
-  {% endfor %}
+    {% endif -%}
+  {% endfor -%}
+</div>
+
+---
+
+## Former Sponsors
+
+<div class="row former">
+  {% for sponsor in site.sponsors -%}
+    {% if sponsor.status == "former" -%}
+    {% include sponsor-card.html width=4 vendor=sponsor %}
```

Comment:
…and use less columns here, e.g.:

```suggestion
    {% include sponsor-card.html width=3 vendor=sponsor %}
```

(untested, IIRC there are 12 columns, so width=4 means 3 logos per line, while width=3 means 4… we can even go to width=2)